### ISSUE TITLE
Deprecation Warning for google-cloud-vision 0.34.0

### ISSUE THREAD

**Issue by nnegrey** [GoogleCloudPlatform/google-cloud-python#5969]
# Thank you for reporting an issue to google-cloud-python!

If you are reporting an issue or requesting a feature, please search the
existing open and closed issues to see if there is already work being done.

- https://github.com/GoogleCloudPlatform/google-cloud-python/issues
- http://stackoverflow.com/questions/tagged/google-cloud-python

If you can provide us with as much of the following information as possible it
will help us identify the cause of your issue more quickly.

1. Specify the API at the beginning of the title (for example, "BigQuery: ...")
- `Google Cloud Vision` 
2. OS type and version
- `Mac OS`
3. Python version and virtual environment information `python --version`
- `Python 3.7.0`
4. google-cloud-python version `pip show google-cloud`, `pip show google-<service>` or `pip freeze`
```
Name: google-cloud-vision
Version: 0.34.0
Summary: Cloud Vision API API client library
Home-page: https://github.com/GoogleCloudPlatform/google-cloud-python
Author: Google LLC
Author-email: googleapis-packages@google.com
License: Apache 2.0
Location: /Users/nnegrey/homebrew/lib/python3.7/site-packages
Requires: google-api-core
Required-by: 
```
5. Stacktrace if available
```
/.../python3.7/site-packages/google/cloud/vision_v1p3beta1/gapic/image_annotator_client.py:119: PendingDeprecationWarning: The `client_config` argument is deprecated.
  PendingDeprecationWarning)
```
6. Steps to reproduce: Running `pytest .`
7. Code example: detect.py 

Using GitHub flavored markdown can help make your request clearer.
See: https://guides.github.com/features/mastering-markdown/

**Comment by sduskis** [comment]
This looks like a duplicate of #5967
